services/121-service/src/scripts/scripts.controller.ts

```diff
@@ -193,6 +199,7 @@ export class ScriptsController {
     summary:
       'WARNING: Kills 121-service. Only works in DEBUG-mode. Only used for testing purposes.',
   })
+  @ApiExcludeEndpoint(!DEBUG)
```

Comment:
nice, this just means it doesn't show up in swagger? or it doesn't exist at all?